### PR TITLE
Add actions and reducers

### PR DESCRIPTION
**Pull request by SergSkol**
-Add redux folder
-Add actions
-Add reducers

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Actions
+const CREATE = 'CREATE';
+const REMOVE = 'REMOVE';
```
Comment (FrederickMih) [inline src/redux/books/books.js]: - [ ] Hi @SergSkol, great job defining CREATE, and REMOVE action types :+1: Now let's kindly make sure that the value of each action type('CREATE') follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)(Ex: `const CREATE = bookstore/books/CREATE`)

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Actions
+const STATUS = 'STATUS';
```
Comment (FrederickMih) [inline src/redux/categories/categories.js]: - [ ] Please would you kindly check the comment above? :arrow_up: Making sure that the values of the action types follow the duck pattern ensures that we can easily scale our app in the future and also facilitate debugging our app when it becomes larger/bulky.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @SergSkol, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


Screenshot from 2022-10-20 09-38-51


## Highlights :green_circle: 

- Nice commit message :+1:
- Fantastic job for utilizing configureStore to create reducers :+1: 
- PR is descriptive :+1: 
- All Linter checks passed :heavy_check_mark: 


## Required Changes ♻️

N/A

_Check the comments under the review._

## Optional suggestions
N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (SergSkol) [reply src/redux/books/books.js]: Thanks! It's done

File: src/redux/categories/categories.js (hunk 2 above)
Comment (SergSkol) [reply src/redux/categories/categories.js]: Thanks! It's done

**Review by levy002 — APPROVED** [review]
## Hello @SergSkol 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**